evals/configs/femnist/conf.yml

```diff
@@ -24,20 +23,20 @@ auth:
 
 # cmd to run before we can indeed run FAR (in order)
 setup_commands:
-    - source $HOME/anaconda3/bin/activate fedscale
+    - source $FEDSCALE_HOME/anaconda3/bin/activate fedscale
```

Comment:
This will be incorrect almost all the time; `conda` may not have been installed in $FEDSCALE_HOME